src/renderer/components/app.tsx

```diff
@@ -69,12 +68,14 @@ import { Nodes } from "./+nodes";
 import { Workloads } from "./+workloads";
 import { Config } from "./+config";
 import { Storage } from "./+storage";
+import { AllowedResources, isAllowedResources } from "../api/allowed-resources";
+import { CubeSpinner } from "./spinner";
 
 @observer
 export class App extends React.Component {
-  constructor(props: {}) {
-    super(props);
-    makeObservable(this);
+  @computed
+  static get startUrl(): string {
```

Comment:
decorators doesn't work with with `static` in `mobx-6`

Reply:
How so? No errors are thrown and it seems to work as intended.

The docs say "Decorating static class members" are not supported. A getter is not a class member.

Reply:
It works as intended just because it's a `getter` :) == just no caching/computations

Reply:
If i got correctly "as intended" == "still observable"?